File: src/main/java/com/github/bgora/american/AmericanCar.java
package com.github.bgora.american;

public record AmericanCar(String name, Integer horsePower, Integer maxSpeed) {

    @Override
    public String toString() {
        return "AmericanCar{" +
                "name='" + name + '\'' +
                ", horsePower=" + horsePower +
                ", maxSpeed=" + maxSpeed + "MPH" +
                '}';
    }
}

File: src/main/kotlin/com/github/bgora/american/AmericanCar.kt
package com.github.bgora.american


data class AmericanCar(val name: String, val horsePower: Int, val maxSpeed: Int) {

    override fun toString(): String {
        return "AmericanCar{" +
                "name='" + name + '\'' +
                ", horsePower=" + horsePower +
                ", maxSpeed=" + maxSpeed + "MPH" +
                '}'
    }
}